test/unit/config.spec.js

```diff
@@ -20,6 +20,30 @@ describe("config", function () {
     cloudinary.config(true);
   });
 
+  it("should support `hide_sensitive` config param", function () {
```

Comment:
Forgot to mention it: I'd add another test case that explicitly proves that the default value for that env var is false.